connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/PlanGenerationTestSuite.scala

```diff
@@ -103,6 +104,12 @@ class PlanGenerationTestSuite
     val client = SparkConnectClient(InProcessChannelBuilder.forName("/dev/null").build())
     session =
       new SparkSession(client, cleaner = SparkSession.cleaner, planIdGenerator = new AtomicLong)
+    if (regenerateGoldenFiles) {
```

Comment:
A simple method is to delete all the corresponding directories when generating golden files. This way, in our logic, we won't accidentally submit redundant files.

Reply:
Good catch! Thank you ! Personally, I feet delete all golden files and regenerate is OK.